spec/visual/samples/hql

```diff
@@ -0,0 +1,5 @@
+SELECT e.first_name, e.last_name, d.department_name
+FROM   employees e
+JOIN   departments d ON e.department_id = d.department_id;
+
+update `table` set name='abc', date=${date}, test_interpolation="${var1}.${var2}" where xyz is null;
```

Comment:
This is a good amount of code for the demo file but for the visual sample, we typically would look for something more extensive. Was your thought that since this lexer largely relies on the SQL lexer, the appropriate thing to test here is the rules that differ?

Reply:
IIRC this was based on what the SQL examples were when I branched this. I agree that more visual documentation is nice, but I'm not sure I'll be able to add that very soon. Would you be okay with merging as-is?